docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1954,9 +1954,10 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/core/disabledOperations`|string|A comma-separated list of operations which are not allowed to run on the IBM FHIR Server, for example, `validate,import`. Note, do not include the dollar sign `$`|
 |`fhirServer/core/defaultPageSize`|integer|Sets the pageSize to use in search and history when no _count parameter is specified in the request. If a user-specified value exceeds the max page size (1000), then a warning is logged and max page size will be used. If not provided, the default page size (10) is used.|
 |`fhirServer/term/disableCaching`|boolean|Indicates whether caching is disabled for the FHIR terminology module, this includes caching in `CodeSystemSupport`, `ValueSetSupport`, `GraphTermServiceProvider`, and `RemoteTermServiceProvider`|
-|`fhirServer/term/graphTermServiceProvider/enabled`|boolean|Indicates whether the graph term service provider should be used by the FHIR term service to access code system content|
-|`fhirServer/term/graphTermServiceProvider/timeLimit`|integer|Graph traversal time limit (in milliseconds)|
-|`fhirServer/term/graphTermServiceProvider/configuration`|object (name/value pairs)|A JSON object that contains the name/value pairs used to configure the graph database behind the graph term service provider see: [https://docs.janusgraph.org/basics/configuration-reference/](https://docs.janusgraph.org/basics/configuration-reference/)|
+|`fhirServer/term/graphTermServiceProviders`|array of objects|The `graphTermServiceProviders` element is an array of objects|
+|`fhirServer/term/graphTermServiceProviders/enabled`|boolean|Indicates whether the graph term service provider should be used by the FHIR term service to access code system content|
```

Comment:
`|`fhirServer/term/graphTermServiceProviders/enabled`` is this really |`fhirServer/term/graphTermServiceProviders/<term-provider>/enabled` 

it's an array, then a boolean in the array?

Reply:
`fhirServer/term/graphTermServiceProviders` is an array of objects and `enabled` is a JSON property within that those objects

Reply:
Sounds fair...

Reply:
Perhaps there is a good way to designate this? maybe using brackets?
`fhriServer/term/graphTermServiceProviders[0..n]/enabled`

Reply:
> @lmsurpre any thoughts

As you know we've avoided arrays of objects to date, so I don't think we have precedence for this.  No strong feelings from me one way or the other, could take or leave the `graphTermServiceProviders[0..n]`.

Reply:
I'm good leaving as is